q2_moshpit/eggnog/tests/test_method.py

```diff
@@ -102,3 +104,26 @@ def test_fetch_eggnog_db(self, subp_run):
             "--data_dir", str(eggnog_db)
         ]
         subp_run.assert_called_once_with(cmd, check=True)
+
+    @patch("subprocess.run")
+    def test_fetch_diamond_db(self, subp_run):
+        # Call function. Patching will make sure nothing is
+        # actually ran
+        diamond_db = fetch_diamond_db()
+
+        # Check that command was called in the expected way
+        cmd = [
+            'echo "Starting download..." && '
+            f'cd {str(diamond_db)} && wget -nH --user-agent=Mozilla/5.0 '
+            '--relative --no-parent --reject "index.html*" --cut-dirs=4 '
+            '-e robots=off -O ref_db.dmnd.gz '
```

Comment:
You can remove most of those flags as they do not really apply here:
- `--user-agent`: impersonating Firefox should not be necessary - the download works without it
- `--relative`, `--no-parent`, `-nH`, `--reject ...` and `--cut-dirs=4` are also not so relevant as you are specifying the output file name directly

Please try without those and only keep the ones that seem to be necessary.

Reply:
Ooop, I meant to make this comment in the actual code... Still, the point remains the same.